src/rh-footer-links-wrapper.ts

```diff
@@ -1,27 +1,30 @@
 import { css, html, LitElement } from 'lit';
 import { state, property } from 'lit/decorators.js';
-import { unsafeHTML } from 'lit/directives/unsafe-html.js';
 import { mobileBreakpoint } from './lib/tokens.js';
+import { MatchMediaController } from './lib/MatchMediaController.js';
 
 interface LinkSet {
   header: HTMLElement | null;
-  panel: Element[];
+  panel: HTMLElement | null;
 }
 
 export class RhFooterLinkWrapper extends LitElement {
   static get tag() {
     return `rh-footer-links-wrapper`;
   }
 
-  @property({ type: Boolean, attribute: 'is-mobile' })
-  public isMobile: boolean = false;
+  @property({ type: Boolean, attribute: 'is-mobile', reflect: true })
+  public isMobile: boolean = new MatchMediaController(
+    this,
+    '(max-width): 600px'
+  ).value;
```

Comment:
this seems like it goes against the grain.

It would be better to not annotate the class field with `property`, and move the `attribute` config to options for `MatchMediaController`, like

```ts
  public isMobile = new MatchMediaController(this, {
    query: '(max-width): 600px',
    attribute: 'is-mobile'
  });
```

or, to move the attribute part to a separate piece

```ts
  // NB: would require that `MatchMediaController` narrow
  // it's acceptable host type to `ReactiveElement` or `LitElement`
  // and call `host.requestUpdate('isMobile', oldValue);
  @observed 
  public isMobile = new MatchMediaController(this, '(max-width): 600px');
  
  protected _isMobileChanged(_, isMobile) {
    this.toggleAttribute('is-mobile', isMobile);
  }
```